src/AppRoot.tsx

```diff
@@ -165,6 +169,15 @@ class AppRoot extends React.Component<RouteComponentProps, State> {
     }
   }
 
+  public fetchVotes = async () => {
+    const response = await fetch('/card/read_long_b64')
+    const returnValue = await response.json()
```

Comment:
This could be `const returnValue = await fetchJSON('/card/read_long_b64')`, ideally with a type parameter for the return object shape.

Reply:
cool

Reply:
I had trouble getting tests to pass with timers, and this seemed to be an additional complication, so I left it as is for now.